apps/timesheet-app/backend/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "wso2"
+name = "sample_app"
+version = "1.0.0"
+distribution = "2201.8.4"
```

Comment:
Let's develop the app using a higher version than this. `2201.11.0` or `2201.10.x`

Reply:
We can update the Ballerina version in the app-specific PR. This is the default template.